cellpack/autopack/upy/simularium/simularium_helper.py

```diff
@@ -1398,23 +1398,35 @@ def raycast_test(self, obj, start, end, length, **kw):
     def post_and_open_file(self, file_name, open_results_in_browser=True):
         simularium_file = Path(f"{file_name}.simularium")
         url = None
+        job_id = os.environ.get("AWS_BATCH_JOB_ID", None)
+        if job_id:
+            print(f"Batch Job ID: {job_id}")
```

Comment:
nit: we can probably remove lines 1402 and 1403, this was just for debugging

Reply:
good catch! Thanks!